src/eachrow.jl

```diff
@@ -95,24 +95,25 @@ end
 """
     @eachrow(d, body)
 
-Act on each row of a data frame, similar to
+Act on each row of a data frame, producing a new dataframe.
+Similar to
 
 ```
-for row in eachrow(df)
+for row in eachrow(copy(df))
     ...
 end
 ```
 
 Includes support for control flow and `begin end` blocks. Since the
-"environment" induced by `@eachrow df` is implicitly a single row of `df`,
+"environment" induced by `@eachrow df` is implicitly a single row of `d`,
```

Comment:
Isn't the name above `df`?
```suggestion
"environment" induced by `@eachrow df` is implicitly a single row of `df`,
```

Reply:
Just made `df` everywhere.

Reply:
In the signature it's `d`, but in the block above it's `df`.

Reply:
Fixed, added `df` to the actual function definition.